### ISSUE TITLE
Add epoll reactor to the choosing-reactor doc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2539]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#2539|
|Type|enhancement|
|Created|2007-03-25 17:45:52Z|

Epoll is great, we should promote it.

<details><summary>Searchable metadata</summary>

```
trac-id__2539 2539
type__enhancement enhancement
reporter__therve therve
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1174844752000000 1174844752000000
changetime__1177315461000000 1177315461000000
version__None None
owner__ 
cc__itamarst
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve removed owner|
|-|-|

Ready to review in epoll-doc-2539.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_2"></a>|@itamarst set owner to @therve|
|-|-|

The only platform that will work is Linux 2.6. Also you should say it's much more scalable than either poll() or select() (the "only a better poll()" isn't that accurate, it's significantly better.)

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve commented|
|-|-|

Oh, I thought it was also available on some BSD. Thanks for the suggestion.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve removed owner|
|-|-|

I've modified the doc according to your review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @therve|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_6"></a>|@therve set status to `closed`|
|-|-|

(In [20042]) Merge epoll-doc-2539

Author: therve
Reviewer: exarkun
Fixes [#2539](https://github.com/twisted/twisted/issues/2539)

Add epoll reactor to the choosing-reactor documentation.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
